### DISCUSSION TITLE
Plex Companion support

### DISCUSSION THREAD

**Discussion by Uncloak2** [pannal/plex-for-kodi#147] [Ideas]
Ability to control PM4K via phone / tablet.

https://support.plex.tv/articles/203082707-supported-plex-companion-apps/

**Comment by pannal** [comment]
I'm waiting to get access to the corresponding implementation APIs.

**Reply by xenonaxel** [reply]
Were you able to get access to the implementation APIs?

**Reply by pannal** [reply]
No, sadly not, sorry.

**Reply by xenonaxel** [reply]
I think the plexkodiconnect dev was able to get it to work?
https://github.com/croneter/PlexKodiConnect/tree/75c2e0ecfe6099501c104fd2a3a064029ee67251/resources/lib/plex_companion
I found this in the codebase over there. Maybe croneter could guide you or assist with documentation.

**Reply by pannal** [reply]
Interesting. Had a peek and seems well laid out and easy enough to understand. Will take a deeper look after 0.8.0-stable.

**Reply by pannal** [reply]
Hmm, I'll try to find out whether the work is worth it. Seems like it's been unsupported officially now? https://support.plex.tv/articles/203082707-supported-plex-companion-apps/

**Reply by xenonaxel** [reply]
I believe the receiver role for Plex Companion is still supported. I installed PlexKodiConnect to confirm it works with Kodi, and it performed flawlessly. It provided a code to authenticate at https://plex.tv/link, which seems to be a distinct characteristic of a Companion-compatible app.

I’ve been able to control and fling media to my Kodi device without any issues. I’m happy to help with testing on my device if you think it would be useful.

**Reply by Uncloak2** [reply]
Any news on this?

**Reply by pannal** [reply]
Sadly the companion API wasn't exposed with the recent API docs release: https://developer.plex.tv/pms/

So not sure how to proceed. I could reverse engineer the API, but at the moment I have bigger fish to fry :/
